components/Dialogs/docs/theming.md

```diff
@@ -31,10 +46,18 @@ MDCAlertControllerThemer.applyScheme(scheme, to: alertController)
 // Step 1: Import the DialogThemer extension
 #import "MaterialDialogs+DialogThemer.h"
 
-// Step 2: Create or get a alert scheme
-MDCAlertScheme *alertScheme = [[MDCAlertScheme alloc] init];
+// Step 2: Create or get a Material container scheme
+MDCContainerScheme *scheme = [[MDCContainerScheme alloc] init];
```

Comment:
This is the code snippet for Themers. Note the import statement above (`+DialogThemer`).  Could you please leave this section as-is (maybe a note that it will be deprecated) and add the Theming docs separately?